### DISCUSSION TITLE
Choose which face is inpainted apart from Detection model confidence threshold

### DISCUSSION THREAD

**Discussion by skeer562** [Bing-su/adetailer#448] [Q&A]
I have 5 faces in my image that have similar confidence levels.  I only want to re-work one of them.  I can't use threshold since that would exclude all of them.  I tried WebUI's inpaint tab and masking the one face, but when I ran adetailer, it still is working to change all the faces (very slow for my settings and card).  How can I get adetailer to only re-work the one face of my choosing?
The bottom left face needs re-touching.  The others look ok.
![00028-2706320942](https://github.com/Bing-su/adetailer/assets/121366673/17914ba9-257a-4b63-922e-c39e260e0d92)

**Comment by CarbonatedPastries** [comment]
If you go to ADetailer's settings, you can choose to `Sort bounding boxes by` and select `Position (left to right)`. This will help make the order more predictable. From there, you can set up the ADetailer prompt using `[SEP]` and `[SKIP]` (more information [here](https://github.com/Bing-su/adetailer/wiki/Advanced)), such as:
```
[SKIP][SEP]PROMPT[SEP][SKIP]
```
Of course adjust the order based on which it selects first; As there are two faces nearly on the same vertical slice, the one you're trying to fix may be first, so you can make that one first instead of skipping it. To not affect the other faces, make sure the last entry is a `[SKIP]`. You don't need to write it for each, as if there are more faces than prompts it reuses the last.

**Comment by LucianoDaluz** [comment]
It can be way easier if there is a option like face,0,1,2,3 and so on.
